controller/metrics/prommetrics.go

```diff
@@ -111,14 +111,14 @@ var (
 	MetricAnalysisRunMetricType = prometheus.NewDesc(
 		"analysis_run_metric_type",
 		"Information on the type of a specific metric in the Analysis Runs",
-		append(namespaceNameLabels, "metric", "type"),
+		append(namespaceNameLabels, "metric", "type", "dryRun"),
 		nil,
 	)
 
 	MetricAnalysisRunMetricPhase = prometheus.NewDesc(
 		"analysis_run_metric_phase",
 		"Information on the duration of a specific metric in the Analysis Run",
-		append(namespaceNameLabels, "metric", "type", "phase"),
+		append(namespaceNameLabels, "metric", "type", "dryRun", "phase"),
```

Comment:
prometheus metrics aren't typically camelCase so I feel this should be `dry_run`

Reply:
Changed, Thanks!